core/metrics/null.go

```diff
@@ -50,6 +52,9 @@ func (n nullHandler) Gin() gin.HandlerFunc {
 func (n nullHandler) Start(_ context.Context) error {
 	return nil
 }
+func (n nullHandler) Meter() *sdkmetric.MeterProvider {
+	return nil
```

Comment:
this isn't gonna work, this'll nil dereference here the second you try to use it:

<img width="547" alt="image" src="https://github.com/synapsecns/sanguine/assets/83933037/62fbdb6a-7a8f-44e9-9297-0fe6b7a5a569">